File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var searchRange = function(nums, target) {
    var ind = [];
    ind.push(-1);
    ind.push(-1);
    if(nums.length==0){
        return ind;
    }
    for(let i = 0; i<nums.length; i++){
        if(nums[i]==target){
            ind[0] = i;
            break;
        }
    }
    if(ind[0]===-1){
        return ind;
    }
    for(let i = ind[0]; i<nums.length; i++){
        if(nums[i]==target){
            ind[1] = i;
        }
    }
    return ind;
};

File: April-Challenges-2021/Day-29/First-and-Last-Position-of-Elements-in-Sorted-Array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        first,second = -1,-1
        if len(nums)==0 or target not in nums:
            return [first,second]
        for i in range(len(nums)):
            if nums[i] == target:
                first = i
                break
        for i in range(first,len(nums)):
            if nums[i] == target:
                second = i
        return [first,second]
